libs/wallet-ui/src/routes/index.tsx

```diff
@@ -23,7 +25,11 @@ export const AppRouter = () => {
     <Routes>
       <Route path="/" element={<Outlet />}>
         <Route index={true} element={<Home />} />
-        <Route path="onboard" element={<Onboard />} />
+        <Route path="onboard" element={<Outlet />}>
```

Comment:
Na we can't because of it starts with a `/` in the enum it breaks the whole router. I've noticed the same thing but I wanted to address as a separate ticket 👍

Reply:
ah, okay, makes sense, didn't see the difference